GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/Podfile

```diff
@@ -14,3 +14,13 @@ target 'GDTCCTWatchOSIndependentTestAppWatchKitExtension' do
   pod 'GoogleDataTransportCCTSupport', :path => '../../'
 
 end
+
+target 'GDTCCTWatchOSCompanionTestAppExtension' do
+  platform :watchos, '6.0'
+
+  # Pods for GDTCCTWatchOSCompanionTestAppExtension
+  pod 'SwiftProtobuf', '~> 1.0'
```

Comment:
Just curious, you guys need this one?

Reply:
This one is only used in test app to generate swift code from proto.